src/Containers/ClientShowCase.tsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Button } from '../Inputs/Button';
+
+const COMPONENT_HEIGHT = '90px'
```

Comment:
We don't add px to strings btw, this is a number so add px to styled components like
const C = 90;
sstyled.View`
height: #{C}px;
`